packages/debs/SPECS/wazuh-agent/debian/changelog

```diff
@@ -1,3 +1,9 @@
+wazuh-agent (5.0.0-RELEASE) stable; urgency=low
+
+  * More info: https://documentation.wazuh.com/current/release-notes/release-4-9-0.html
```

Comment:
Are these lines necessary or correct the way they are? I wonder if we should tag this as `alpha` instead. And wonder if there's an alternative to stable, since we are in such early stages of development.

Reply:
This line is necessary for the package to appear as a `5.0.0` package, otherwise it would appear as a `4.9.0` package (because it is the last line in the changelog). And the tag can be specified directly when building the package, while the changelog has nothing to do with it.